source/sonata_overview.rst

```diff
@@ -43,17 +43,28 @@ A standard circuit config at |bbp| is :
         "nodes": [
           {
             "nodes_file": "$NETWORK_DIR/neurons.h5",
-            "node_types_file": "$NETWORK_DIR/node_types.csv"
+            "node_types_file": null,
```

Comment:
we can omit this since it is optional.
since the sentence says 'A standard circuit config at BBP is...